frontend/src/conf.js

```diff
@@ -0,0 +1,9 @@
+require("dotenv").config();
+
+const backend = process.env.REACT_APP_BACKEND;
+console.log(process.env.REACT_APP_BACKEND)
```

Comment:
Damnit, NO !

Reply:
Oups ^^ deleted

> Please add the `backend/tmp` folder to your `.gitignore`
> Or better yet, stop using sessions while working with authentication !

I'm not sure of the origin of this tmp folder. Authentification is not on this branch yet.
I think this tmp folder is from the branch to upload file on cloudinary which is on a local repo.
I have added /tmp to the gitignore but it's still send to git.